projectroles/project_tags.py

```diff
@@ -1,59 +1,24 @@
 """Functions for project tagging/starring in the projectroles app"""
 # NOTE: This can be expanded to include other types of tags later on
 
-from projectroles.models import ProjectUserTag, PROJECT_TAG_STARRED
+from projectroles.app_settings import AppSettingAPI
 
 
-def get_tag_state(project, user, name=PROJECT_TAG_STARRED):
-    """
-    Get current starring status of a project/user.
+app_settings = AppSettingAPI()
 
-    :param project: Project object
-    :param user: User object
-    :param name: Tag name (string)
-    :return: Boolean
-    """
-    if not user.is_authenticated:
-        return False
 
-    try:
-        ProjectUserTag.objects.get(project=project, user=user, name=name)
-        return True
+def get_tag_state(project, user):
```

Comment:
Also, we can just call `app_settings.set()` and `get()`, no need for further helpers I think.